committime_template/action.yml

```diff
@@ -0,0 +1,37 @@
+name: "Commit time action"
+description: "Composite action for Commit time metric for Pelorus."
+
+inputs:
+  app:
+    description: "App to monitor"
+    required: true
+  url:
+    description: "Url of pelorus endpoint"
+    default: http://webhook-exporter-pelorus.apps.grey.dev.iad2.dc.paas.redhat.com/pelorus/webhook
```

Comment:
I would make this field required as well

Reply:
Fully agree. This is something that needs to be a parameter, not a hardcoded value.

Also don't forget to change this for the other action template.

Reply:
Good point. Changed